content/docs/sidebar.json

```diff
@@ -620,7 +620,6 @@
         "slug": "dvclive-with-dvc"
       },
       {
-        "label": "ML Frameworks",
```

Comment:
Since the sidebar code automatically uppercases `ml` and `api`, these two labels can be taken off.